client/components/tinymce/plugins/insert-menu/plugin.jsx

```diff
@@ -15,8 +15,8 @@ const initialize = editor => {
 			cmd: item.cmd,
 			onPostRender() {
 				this.innerHtml( renderToString( item.item ) );
-			}
-		} )
+			},
+		} ),
```

Comment:
It seems that you don't have an up-to-date version of `calypso-prettier`: we recently changed the value of the `trailingComma` option from `all` to `es6`. There shouldn't be a trailing comma after the last function arg:
```js
each(
  a,
  b // no comma
);

each( {
  a: 1,
  b: 2, // yes comma
} );
```